pkg/projects/resource_project_user.go

```diff
@@ -0,0 +1,214 @@
+package projects
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/go-resty/resty/v2"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+type User struct {
+	Name  string   `json:"name"`
+	Roles []string `json:"roles"`
+}
+
+type Comparable interface {
+	Hash() string
+	Equals(other Comparable) bool
+}
+
+func (user User) Hash() string {
+	return user.Name
+}
+
+func (a User) Equals(b User) bool {
+	return a.Hash() == b.Hash()
+}
+
+func contains(as []User, b User) bool {
+	log.Printf("[DEBUG] contains")
+	log.Printf("[TRACE] as: %+v\n", as)
```

Comment:
This is a lot of logging - a debugger would eliminate it all

Reply:
I hate debugger

Reply:
These don't show up unless env var `TF_LOG` is set to the level, e.g. `export TF_LOG=DEBUG`. So you get to choose what you want to see 😄